src/common/Utility.js

```diff
@@ -7,6 +7,27 @@
     }
 }(this, function (d3) {
 
+    function hexColorConv(_) {
+        if (_.length === 7) {
+            return _;
+        } else if (_ === "") {
+            return null;
+        }
+        if (_.charAt(0) === "#") { 
+            return "#" + _
+                .split("")
+                .map(function(c, i) {
+                    if (c === "#" && i === 0) {
+                        return;
+                    }
+                    return c+c; 
+                })
+                .join("")
+            ;
+        }
+        return _;
+    }
+
```

Comment:
Not needed, should use d3 color helpers:  https://github.com/mbostock/d3/wiki/Colors